core/pom.xml

```diff
@@ -39,7 +39,7 @@ THE SOFTWARE.
 
   <properties>
     <staplerFork>true</staplerFork>
-    <stapler.version>1.253</stapler.version>
+    <stapler.version>1.254-SNAPSHOT</stapler.version>
```

Comment:
staged release would be nice, but IIUC Stapler settings need to be patched for that

Reply:
This doesn't result in a consistent date for every artifact, which is the reason I used `-SNASPSHOT` rather than changing all the uses of `stapler.version`.

Reply:
1.254 released with just this change, please update.